addons/website_sale/static/src/js/product_ribbon_mixin.js

```diff
@@ -0,0 +1,285 @@
+import { ConfirmationDialog } from "@web/core/confirmation_dialog/confirmation_dialog";
+import { renderToElement } from "@web/core/utils/render";
+import { _t } from "@web/core/l10n/translation";
```

Comment:
Could you change to single quotes please?
```suggestion
import { ConfirmationDialog } from "@web/core/confirmation_dialog/confirmation_dialog";
import { renderToElement } from "@web/core/utils/render";
import { _t } from "@web/core/l10n/translation";
```